### PR TITLE
upgrade to assertj-core 3.17.2

### PR DESCRIPTION
**Pull request by sullis**
AssertJ 3.17.2 is the latest version of assertj-core

### CODE REVIEW

**Review by pivovarit — APPROVED** [review]
Thanks! It looks like a lot of annoying work!

Waiting for @mchmielarz to have a look as well

**Comment by sullis** [comment]
ping @mchmielarz

File: src/main/java/org/assertj/vavr/api/AbstractMultimapAssert.java [hunk 1]
```diff
@@ -18,12 +18,17 @@
...
+import static org.assertj.core.error.ShouldHaveSizeGreaterThan.shouldHaveSizeGreaterThan;
+import static org.assertj.core.error.ShouldHaveSizeGreaterThanOrEqualTo.shouldHaveSizeGreaterThanOrEqualTo;
+import static org.assertj.core.error.ShouldHaveSizeLessThan.shouldHaveSizeLessThan;
+import static org.assertj.core.error.ShouldHaveSizeLessThanOrEqualTo.shouldHaveSizeLessThanOrEqualTo;
 import static org.assertj.core.error.ShouldNotBeEmpty.shouldNotBeEmpty;
 import static org.assertj.core.util.Arrays.array;
 import static org.assertj.core.util.IterableUtil.sizeOf;
 import static org.assertj.core.util.Preconditions.checkNotNull;
 
-class AbstractMultimapAssert<SELF extends AbstractMultimapAssert<SELF, ACTUAL, KEY, VALUE>, ACTUAL extends Multimap<KEY, VALUE>, KEY, VALUE>
```
Comment (mchmielarz) [inline src/main/java/org/assertj/vavr/api/AbstractMultimapAssert.java:26]: 👍

**Review by mchmielarz — COMMENTED** [review]
I see no tests checking size greater/less/equal assertions. Could you add them?

**Comment by sullis** [comment]
> I see no tests checking size greater/less/equal assertions. Could you add them?

I have added additional tests:
- MultiMapAssert_isNotEqualTo_Test
- MapAssert_isNotEqualTo_Test
- MapAssert_hasSizeGreaterThan_Test
- MapAssert_hasSizeLessThan_Test
- MapAssert_isEqualTo_Test
- MultimapAssert_hasSizeGreaterThan_Test
- MultimapAssert_hasSizeLessThan_Test
- MultiMapAssert_isEqualTo_Test

**Comment by sullis** [comment]
@pivovarit @mchmielarz I have no further work planned. Can we merge this PR?

**Comment by mchmielarz** [comment]
Good work! 💯 Thank you!

**Pull request merged**